Code/Framework/AzCore/AzCore/Script/ScriptSystemComponent.cpp

```diff
@@ -371,6 +371,12 @@ ScriptLoadResult ScriptSystemComponent::LoadAndGetNativeContext(const Data::Asse
             }
             else
             {
+                SCRIPT_SYSTEM_SCRIPT_STATUS("ScriptSystemComponent"
+                    , "ScriptSystemComponent::LoadAndGetNativeContext returning pre-loaded script: %s-%s"
+                    , asset.GetHint().c_str()
+                    , asset.GetId().ToString<AZStd::string>().c_str());
```

Comment:
FYI you can avoid the a heap allocation when converting a Uuid to a string by usign
[AZStd::fixed_string<AZ::Uuid::MaxStringBuffer>](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Math/Uuid.h#L46) instead

```suggestion
                    , asset.GetId().ToString<AZStd::fixed_string<AZ::Uuid::MaxStringBuffer>>().c_str());
```

Reply:
I will fix that up immediately after this review, but by request I've been asked to not let it grow more than it already has.

Reply:
Changing a line is a one-to-one trade, the review doesn't grow in that case.

Reply:
Done